docs/RELEASE_NOTES.md

```diff
@@ -146,287 +146,277 @@ Use Cocos Console to create a new game:
 
-[REFINE]        Tool: Added tools for generating documents automatically.
+[TEST]          S9SpriteTest: Scale9Sprite fade actions with cascade opacity.
 
-[REFINE]        Doc: Clean up the code of setRect() function.
+[TEST]          Web: Removed default focus block from UIFocusTestVertical.
 
-[REFINE]        Doc: Fixed a minor typo and renamed INTIAL_CAPS_ALL_CHARACTERS to INITIAL_CAPS_ALL_CHARACTERS 
-in UIEditBox.
+[TEST]          Lua: Fixed pageViewTest Horizontal scroll won't work in Lua-test.
 
-You can also take a look at the [full changelog](https://github.com/cocos2d/cocos2d-x/blob/v3/CHANGELOG).
+You can also take a l
+ook at the [full changelog](https://github.com/cocos2d/cocos2d-x/blob/v3/CHANGELOG).
 
 
 ## NEW APIS
 
-1. JSB Module
+1. RichText
+
+    Added RichElementNewLine class to create new RichText Element.
+
+    For more information: [https://github.com/cocos2d/cocos2d-x/pull/14033](https://github.com/cocos2d/cocos2d-x/pull/14033 "https://github.com/cocos2d/cocos2d-x/pull/14033")
+
+2. PageViewIndicator
+
+    Added PageViewIndicator class to create PageViewIndicator.
+
+    For more information: [https://github.com/cocos2d/cocos2d-x/blob/v3/cocos/ui/UIPageViewIndicator.h](https://github.com/cocos2d/cocos2d-x/blob/v3/cocos/ui/UIPageViewIndicator.h "https://github.com/cocos2d/cocos2d-x/blob/v3/cocos/ui/UIPageViewIndicator.h")
+
+3. PageView 
+
+    Changed PageView to derived from ListView.
+    For more information: [https://github.com/cocos2d/cocos2d-x/pull/14252](https://github.com/cocos2d/cocos2d-x/pull/14252 "https://github.com/cocos2d/cocos2d-x/pull/14252")
+
+4. ApplicationProtocol
+
+    Added the API getVersion.
+
+
+5. PolygonInfo
+
+    Added the API setTriangles.
+
+6. Scale9Sprite
+
+    Added adjustNoneScale9ImagePosition, configureSimpleModeRendering, calculateUV, calculateVertices, calculateTriangles, setRenderingType, getRenderingType.
+
+7. CWin32InputBox
+
+    Added GetMsgProc, onWin32InputBoxClose.
 
-    Added jsb.fileUtils.writeDataToFile
+8. FontAtlas
 
-2. Sprite3D
+    Added scaleFontLetterDefinition.
 
-    Added Sprite3Dmaterial class to make it easy to create innate material. 
+9. FontFNT
 
-3. ActionTimelineData 
+    Added setFontSize, getOriginalFontSize.
 
-    Three interfaces are added in ActionTimelineData class, which are addFrameEndCallFunc, removeFrameEndCall and clearFrameEndCalls.
+10. Label
 
-4. ActionTimeline::removeFrameEndCallFunc 
+    Added setBMFontSize, getBMFontSize, enableWrap, isWrapEnabled, setOverflow, getOverflow, multilineTextWrap, shrinkLabelToContentSize, isHorizontalClamp, isVerticalClamp, getRenderingFontSize, rescaleWithOriginalFontSize, updateBMFontScale, scaleFontSizeDown, setTTFConfigInternal, setBMFontSizeInternal, restoreFontSize, updateLetterSpriteScale, initWithTTF, isShadowEnabled, getShadowOffset, getShadowBlurRadius, getShadowColor, getOutlineSize, getLabelEffectType, getEffectColor.
```

Comment:
```multilineTextWrap, shrinkLabelToContentSize, isHorizontalClamp, isVerticalClamp, getRenderingFontSize, rescaleWithOriginalFontSize, updateBMFontScale, scaleFontSizeDown, setTTFConfigInternal, setBMFontSizeInternal, restoreFontSize, updateLetterSpriteScale, ```

These methods are private. I think you should double check only the **public APIs**.